### DISCUSSION TITLE
NVENC AV1 publishing from OBS, supported?

### DISCUSSION THREAD

**Discussion by hiinaspace** [bluenviron/mediamtx#1393] [General]
Has anyone got AV1 publishing to work? My friend recently got an nvidia 4x card, for which OBS supports AV1 hardware encoding. However, the publish (over RTMP) fails with 

    (unable to parse H264 config: sps count != 1 is unsupported)

in the logs, with a config that otherwise works with nvenc h264 or even HEVC just fine.

**Comment by aler9** [comment]
Hello, this is not supported yet since AV1 encoding has been introduced in OBS 29, which was released less than a month ago, and only with certain hardware, starting from the RTX 30 series. I have a RTX 3090 and i'll surely make a couple of tests.

I've created a feature request in order to track the development: #1446
